scala_proto/scala_proto.bzl

```diff
@@ -414,7 +414,7 @@ Example:
 
 Args:
     name: A unique name for this rule
-    deps: Proto library targets that this rule depends on (must be of type proto_library)
+    deps: Proto library targets or jvm targets that this rule depends on
     with_grpc: Enables generation of grpc service bindings for services defined in deps
     with_java: Enables generation of converters to and from java protobuf bindings
```

Comment:
to do this you must also depend on the `java_proto_library`? Is that right? Should we update this comment?